pyspider/webui/static/index.css

```diff
@@ -106,3 +106,26 @@ h1 {
   margin-top: -5px;
   padding: 0 60px 10px 10px;
 }
+.project-create-my .label{
```

Comment:
It's not appropriate edit .css here, the .css is automatically generated via [.less](http://lesscss.org/). The changes you have made here would be overwrite.